### ISSUE TITLE
[LETTERS][a11y-defect-3]: Confirm your address alert has an incorrect call to action that should be updated for clarity

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#111596]
### Point of contact

@1copenut

### Severity level

3, Moderate. Should be fixed in 2-3 sprints post-launch.

### Details

While auditing the Your VA Letters and Documents, I noticed our "Confirm your address" alert includes the sentence "Or, cancel to edit the address." as the last line. Our new workflow doesn't allow users to cancel their editing at this view, so we should rewrite this sentence to direct users to edit their address again. Screenshot attached below.

---
![Image](https://github.com/user-attachments/assets/4630c4d7-10c7-4970-a8ee-ab42f6bca207)

### Reproduction steps

1. Log into Staging with a test user. I've been testing with user 54.
2. Navigate to the letters app, turn on the screen reader of your choosing
3. Click "Edit your address" and add a nonsense value like "#55" to the Street Address Line 2 field
4. Save your address edit
5. Confirm the yellow "Confirm your address" alert appears

### Acceptance criteria
- [ ] New message is clear to sighted and non-sighted users what their next step(s) should be

### References, articles, or WCAG support

1. [Understanding SC 1.3.1: Info and Relationships Level A)](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)
1. [Understanding SC 3.3.4: Error Prevention (Legal, Financial, Data) (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/error-prevention-legal-financial-data.html)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Voice command
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [x] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
This may be something that needs to be updated by the profile team; there is some in flight work on making updates to address validation.  Let's see what's going on with that and see which team will end up owning this piece.
